addon/index.js

```diff
@@ -136,7 +136,7 @@ export function customElement() {
 
 /**
  * Gets the custom element node for a component or application instance.
- * 
```

Comment:
Sorry about all this whitespace removal in here; normally I try to avoid committing these fixed but I missed it this time. I can rebase and remove these from the diff if you want.

Reply:
Nah, it's appreciated, but you don't have to go through that effort. :)

Yes, that's something I may introduce in the near future.